android/src/paper/java/com/swmansion/reanimated/NativeReanimatedModuleSpec.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * This code was generated by [react-native-codegen](https://www.npmjs.com/package/react-native-codegen).
+ *
+ * Do not edit this file as changes may cause incorrect behavior and will be lost
+ * once the code is regenerated.
+ *
+ * @generated by codegen project: GenerateModuleJavaSpec.js
+ *
+ * @nolint
+ */
+
```

Comment:
Is this file really generated by react-native-codegen or just copied from somewhere?

Reply:
It is copied from the `build` folder generated by codegen when using new arch. We don't change those files for better diffs is we changed anything in the spec.